Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/CapsuleViewportEdit.cpp

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/ComponentModes/CapsuleViewportEdit.h>
+#include <AzToolsFramework/ComponentModes/BoxViewportEdit.h>
+#include <AzFramework/Viewport/ViewportColors.h>
+#include <AzFramework/Viewport/ViewportConstants.h>
+
+namespace AzToolsFramework
+{
+    namespace
+    {
```

Comment:
Why an empty namespace here?

Reply:
I lifted this from the original code in the PhysX collider component mode. I think it's to confine those constants to this file only.

Reply:
yeah anonymous namespaces provide internal linkage (like adding `static` in the .cpp, it's more modern C++ style than using `static` but I normally forget)